### DISCUSSION TITLE
need help

### DISCUSSION THREAD

**Discussion by newuser3456** [dnschneid/crouton#4988] [General]
```
```hi new to linux and just wondering if it possible to get the steps needed to install linux. i am not sure if would download code a and just downlad the zip file or there some other file i need 

thanks

**Comment by CroutonIsFun** [comment]
[First read this](https://github.com/dnschneid/crouton)

When crouton has been installed, try this:

`sudo CROUTON_BRANCH=silence crouton -r xenial - xfce`

The issues tab is not for the purpose of teaching newbies how to use Linux. It is for reporting errors in crouton itself. You could try the discussions page. Also, read everything you can about Linux in general.

**Comment by DennisLfromGA** [comment]
@newuser3456, @CroutonIsFun,

Here's a slight change to the above command, it needs the `-t` option for the 'target':

- `sudo CROUTON_BRANCH=silence crouton -r xenial -t xfce`

-**DennisLfromGA**
